src/Servers/Kestrel/Core/src/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -108,7 +108,10 @@ public static unsafe string GetHeaderName(this Span<byte> span)
             return asciiString;
         }
 
-        public static unsafe string GetAsciiStringNonNullCharacters(this Span<byte> span)
+        public static string GetAsciiStringNonNullCharacters(this Span<byte> span)
```

Comment:
Why do we need both the span and readonly span versions of these?

Reply:
Was trying to minimize code churn. The HTTP1 parser uses Span in many places, where HTTP2 uses ReadOnlySpan most of the time.

Reply:
kk that's fine.